components/menu/calendar/Screens/LoginScreen/index.js

```diff
@@ -25,6 +26,7 @@ export default class LoginScreen extends Component {
            .signInWithCredential(credential).then(() => {
            })
            .catch((error) => {
+             // eslint-disable-next-line no-alert
              alert(error);
            // ...
```

Comment:
Would alerting the user by displaying the error be useful to them ?

Reply:
No, removed